sovtokenfees/doc/Interface/README.md

```diff
@@ -377,3 +377,125 @@ Example:
+        * lookup through config state for getting amount of "add_new_steward" alias (_`from plugin's side`_)
+        * making can_pay_fees validation (_`from plugin's side`_)
+        
+### Notes:
+* The order of previous steps is very important. First of all SET_FEES is required, then - AUTH_RULE.
+* SET_FEES is "updating" transaction, so that it appends new aliases to the existing FEEs map (either adding or overriding aliases). For example, if current fees are {A: 1, B: 2} then after sending SET_FEES transaction with {A: 42, C:3}, the resulted map will look like {A: 42, B: 2, C:3}. 
+* Setting fees without adding or changing metadata in corresponding auth_rule doesn't have any effect.
+        
+## How to setup fees for whole pool
+For setting fees for whole we need to make the following steps:
+* Define all the actions which we would like to set fees for
+* Repeat all the steps from [How to set fees for an action](#how-to-set-fees-for-an-action) for each action
+
+## How to change fees amount for alias
+For changing amount of fees for existing alias, you need to send a SET_FEES (as described in [How to set fees for an action](#how-to-set-fees-for-an-action)) transaction with 'fees' value, like:
+```
+{<str: fees alias to change>: <int: new amount>}
+```
+As was mentioned before, SET_FEES is "updating" transaction and this request will update whole fees map in state and new amount of fees_alias will be used for validation.
+Full SET_FEES request:
+ ```
+{
+    "reqId": <int>,             //random identifier
+    "protocolVersion": <int>,   // the version of the client/node communication protocol
+    "operation": {
+        "type": "20000",
+        "fees": {
+            <str: fees alias to change>: <int: new amount>,
+        }
+    },
+}
+```
+
+## How to set fees for complex Auth Constraints
+For example, we have a constraint like:
+```
+(TRUSTEE, 2) or (STEWARD, 5)
+```
+It means, that this action requires "2 signatures from 2 different TRUSTEEs" or "5 signatures from 5 different STEWARDs" and we want to set fees for steward's part of this constraint.
+For this case, we need to:
+* add new alias, 'some_action_by_steward' for example, as described in [How to set fees for an action](#how-to-set-fees-for-an-action)
+* set this alias for corresponding auth_rule into steward's part of constraint:
+```
+(TRUSTEE, 2) or (STEWARD, 5, 'fees': 'some_action_by_steward')
+```
+After that, the next requests will be ordered:
+* with 2 and more TRUSTEE's signatures
+* with 5 and more STEWARD's signature and required amount of fees.
+
+Also, trustee's part of constraint can contains 'fees' field with different fees alias, like:
+```
+(TRUSTEE, 2, 'fees': 'some_action_by_trustee') or (STEWARD, 5, 'fees': 'some_action_by_steward')
+``` 
+'some_action_by_trustee' should exists before setting it in AUTH_RULE transaction.
+
+## Recommendation for setting fees.
+* **Amount of fees for trustees_fees alias doesn't equal to amount of fees for steward_fees.**
```

Comment:
`Amount of fees for trustees_fees alias doesn't equal to amount of fees for steward_fees` - this is not a separate recommendation, but just a clarification for the example in the next recommendation.